test/puppetlabs/puppetdb/integration/smoke.clj

```diff
@@ -27,3 +28,94 @@
           (is (= catalog-uuid report-uuid)))))))
 
 
+(when test-rich-data?
+  (def rich-data-tests
```

Comment:
Are we holding off on testing the alien objects for the time being? I just noticed them in the doc but I'm not sure what they are exactly and if we need to be looking out for them.

Reply:
Yeah we are waiting on that (but I need to open a ticket to add those), they would require also loading some ruby code to bootstrap those objects, which we don't have the infrastructure for in place right now